aspnetcore/mvc/views/tag-helpers/built-in/environment-tag-helper.md

```diff
@@ -3,55 +3,62 @@ title: Environment Tag Helper in ASP.NET Core
 author: pkellner
 description: ASP.NET Core Environment Tag Helper defined including all properties
 ms.author: riande
-ms.date: 07/14/2017
+ms.custom: mvc
+ms.date: 10/10/2018
 uid: mvc/views/tag-helpers/builtin-th/environment-tag-helper
 ---
 # Environment Tag Helper in ASP.NET Core
 
 By [Peter Kellner](http://peterkellner.net) and [Hisham Bin Ateya](https://twitter.com/hishambinateya)
 
-The Environment Tag Helper conditionally renders its enclosed content based on the current hosting environment. Its single attribute `names` is a comma separated list of environment names, that if any match to the current environment, will trigger the enclosed content to be rendered.
+The Environment Tag Helper conditionally renders its enclosed content based on the current [hosting environment](xref:fundamentals/environments). The Environment Tag Helper's single attribute, `names`, is a comma-separated list of environment names. If any of the provided environment names match the current environment, the enclosed content is rendered.
+
+For an overview of Tag Helpers, see <xref:mvc/views/tag-helpers/intro>.
 
 ## Environment Tag Helper Attributes
 
 ### names
 
-Accepts a single hosting environment name or a comma-separated list of hosting environment names that trigger the rendering of the enclosed content.
+`names` accepts a single hosting environment name or a comma-separated list of hosting environment names that trigger the rendering of the enclosed content.
 
-These value(s) are compared to the current value returned from the ASP.NET Core static property `HostingEnvironment.EnvironmentName`.  This value is one of the following: **Staging**; **Development** or **Production**. The comparison ignores case.
+Environment values are compared to the current value returned from the ASP.NET Core static property [IHostingEnvironment.EnvironmentName](xref:Microsoft.AspNetCore.Hosting.IHostingEnvironment.EnvironmentName*). The comparison ignores case.
```

Comment:
Environment values are compared to the current value returned from ~the ASP.NET Core static property~ [IHostingEnvironment.EnvironmentName](xref:Microsoft.AspNetCore.Hosting.IHostingEnvironment.EnvironmentName*)